### DISCUSSION TITLE
Error code 28

### DISCUSSION THREAD

**Discussion by stephen-14** [ablab/spades#911] [Q&A] [answered]
Hello, I've tried to run my assembly with SPAdes 3.12.0 during processing I faced with problem #28. I check the memory of my PC is still enough. 
Thanks for your help!
[params.txt](https://github.com/ablab/spades/files/8039077/params.txt)
[spades.log](https://github.com/ablab/spades/files/8039079/spades.log)

**Comment by asl** [comment, marked as answer]
@stephen-14 Please use Issues for these kind of problems. I created https://github.com/ablab/spades/issues/912 for you. The issue could be clearly seen from the log, as you do not have enough disk free space

**Reply by stephen-14** [reply]
I'll check the memory and try it again. Thank you for your replying :))
